### ISSUE TITLE
couple mobile issues post-#91

### ISSUE THREAD

**Issue by paulswartz** [codeforboston/finda#95]
![finda](https://f.cloud.github.com/assets/214921/2467228/e00b1122-afba-11e3-8d49-e49d24b3a75b.png)

**Comment by paulswartz** [comment]
The 'About' page is a bit wonky as well.

**Comment by WheresHJ** [comment]
Yup, I'll address all that when I do the mobile CSS. Let's catch anything else that seems busted here.
